cla_frontend/apps/cla_provider/urls.py

```diff
@@ -20,6 +22,7 @@
         provider_views.legal_help_form,
         name='legal_help_form'
     ),
+    url(r'^zendesk/$', ZendeskView.as_view(), name='zendesk'),
```

Comment:
You need to add ``cla_provider_zone_required`` to the view so that only logged-in specialists can access it.


    from cla_auth.utils import cla_provider_zone_required
    
    url(r'^zendesk/$', cla_provider_zone_required(ZendeskView.as_view()), name='zendesk'),